test/e2e/data_setup/schema/recordset/product-recordset.json

```diff
@@ -825,7 +869,165 @@
           "display": "Categories"
         },
         "tag:isrd.isi.edu,2016:visible-columns": {
-          "*": ["id", "term"]
+          "*": ["id", "term", "type"]
+        },
+        "tag:isrd.isi.edu,2016:visible-foreign-keys" : {
+          "*" : [
+              ["product-recordset", "fk_facilities"]
+          ]
+        }
+      }
+    },
+    "facilities": {
+      "comment": "Type of facilities ('Luxury/Upscale/Basic')",
+      "kind": "table",
+      "keys": [
+        {
+          "comment": "vocabulary",
+          "annotations": {},
+          "unique_columns": [
+            "id"
+          ]
+        },
+        {
+          "comment": "vocabulary",
+          "annotations": {},
+          "unique_columns": [
+            "type"
+          ]
+        }
+      ],
+      "entityCount": 0,
+      "foreign_keys": [],
+      "table_name": "facilities",
+      "schema_name": "product-recordset",
+      "column_definitions": [
+        {
+          "comment": null,
```

Comment:
For future reference, there's no need to include this property if it's not being used.